src/buckets/components/createBucketForm/MiniFileDnd.tsx

```diff
@@ -158,6 +162,8 @@ export const MiniFileDnd: FC<Props> = ({
     inputEl.current.click()
   }
 
+  // maybe add a title or popover to the left....talk to david kim
```

Comment:
✂️ this comment, but talk to David first :)

Reply:
he's on vacation for a week!  will talk to him when he gets back

Reply:
(i put a comment on the epic to talk to him; concerned that an email or a slack would get lost)